WeakAuras/WeakAuras.toc

```diff
@@ -50,6 +54,7 @@ Animations.lua
 Conditions.lua
 
 # Trigger systems
+LibSpecializationWrapper.lua
```

Comment:
LibSpecializationWrapper.lua should only be loaded on retail

otherwise `1x WeakAuras\LibSpecializationWrapper.lua:4: Cannot find a library instance of "LibSpecialization".`